modules/core/src/main/scala/com/azavea/stac4s/extensions/layer/LayerItemExtension.scala

```diff
@@ -0,0 +1,25 @@
+package com.azavea.stac4s.extensions.layer
+
+import com.azavea.stac4s.extensions.ItemExtension
+
+import cats.Eq
+import io.circe.{Decoder, Encoder}
+import io.circe.syntax._
+
+case class LayerItemExtension(ids: List[String])
+
+object LayerItemExtension {
+  implicit val eqLayerProperties: Eq[LayerItemExtension] = Eq.fromUniversalEquals
+
+  implicit val encLayerProperties: Encoder.AsObject[LayerItemExtension] =
+    Encoder.AsObject.instance[LayerItemExtension] { o => Map("layer:ids" -> o.ids.asJson).asJsonObject }
+
+  implicit val decLayerProperties: Decoder[LayerItemExtension] = Decoder[Map[String, List[String]]] emap {
+    _.get("layer:ids") match {
+      case Some(l) => Right(LayerItemExtension(l))
+      case _       => Left("Could not decode LayerProperties.")
+    }
+  }
```

Comment:
My bad, it is a not working decoder; it should be smth like:

```scala
implicit val decLayerProperties: Decoder[LayerItemExtension] = Decoder.decodeJsonObject emap {
    _.toMap.get("layer:ids") match {
      case Some(l) => l.as[List[String]].map(LayerItemExtension.apply).leftMap(_ => "Could not decode LayerProperties.")
      case _       => Left("Could not decode LayerProperties.")
    }
  }
```

Reply:
Can you say a little bit more about what went wrong here? I'm having trouble figuring out what the test you suggest below should show from looking at this.

Reply:
The decoder itself is wrong, add the following unit test to test it (insert the line below [here](https://github.com/azavea/stac4s/blob/master/modules/core/src/test/scala/com/azavea/stac4s/CatalogLayerSpec.scala#L321): 

```
ItemExtension[LayerItemExtension].getProperties(item) shouldBe LayerItemExtension(List("layer-us", "layer-ca")).valid
```

Reply:
6dcbdc1